confignode/src/main/java/org/apache/iotdb/confignode/manager/IManager.java

```diff
@@ -629,9 +631,11 @@ TDataPartitionTableResp getOrCreateDataPartition(
 
   TGetRegionIdResp getRegionId(TGetRegionIdReq req);
 
-  TGetTimeSlotListResp getTimeSlotList(GetTimeSlotListPlan plan);
+  TGetTimeSlotListResp getTimeSlotList(TGetTimeSlotListReq req);
 
-  TGetSeriesSlotListResp getSeriesSlotList(GetSeriesSlotListPlan plan);
+  TCountTimeSlotListResp countTimeSlotList(TCountTimeSlotListReq req);
+
+  TGetSeriesSlotListResp getSeriesSlotList(TGetSeriesSlotListReq req);
```

Comment:
Add function annotations for these interfaces

Reply:
added